lib/components/Charts/AreaChart/index.stories.tsx

```diff
@@ -21,6 +21,8 @@ const meta: Meta<typeof AreaChart<typeof dataConfig>> = {
     },
     yAxis: {
       hide: true,
+      tickFormatter: (value: string) =>
+        `${Number.isNaN(parseFloat(value)) ? value : (parseFloat(value) / 100).toFixed(2) + "€"}`,
```

Comment:
I'm not sure we should fix this behaviour, for salaries for example, we don't need decimal positions.

Reply:
This is only for storybook as an example and by default is hided this yAxis, so you will not see that at first.